core-service/src/main/java/com/crmfoodestablishment/coreservice/entity/Dish.java

```diff
@@ -0,0 +1,29 @@
+package com.crmfoodestablishment.coreservice.entity;
+
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
```

Comment:
I heared that @Builder is not reccomend with @AllArgsConstructor, and @AllArgsConstructor bad practice in Entity 
Mb @NoArgsConstructor @AllArgsConstructor wiil replace on @RequiredArgsConstructor

Reply:
i don't see any issues with use of @AllArgs.. @NoArgs...